### PR TITLE
Code refactory and UI improvements for CompanionPane sample

### PR DESCRIPTION
**Pull request by ancirja-m**

### CODE REVIEW

File: CompanionPane/src/main/res/layout-land/dual_screen_layout_end.xml [hunk 1]
```diff
@@ -4,60 +4,54 @@
...
         android:layout_height="wrap_content"
-        android:background="@color/white"
-        android:minHeight="?attr/actionBarSize"
-        android:layout_marginBottom="@dimen/large_padding" />
+        android:layout_gravity="center"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
-    <LinearLayout
```
Comment (bimiron) [inline CompanionPane/src/main/res/layout-land/dual_screen_layout_end.xml:25]: I like the flattening of the hierarchy, but can we make some changes to keep the center position of the controls? I feel like it looks a bit off with so much space at the bottom

Comment (ancirja-m) [reply CompanionPane/src/main/res/layout-land/dual_screen_layout_end.xml:25]: Done

**Review by bimiron — APPROVED** [review]

File: CompanionPane/src/main/res/layout/effect_control.xml [hunk 2]
```diff
@@ -4,149 +4,135 @@
 ~  Licensed under the MIT License.
 ~
 -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:id="@+id/effect_control"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (CesarValiente) [inline CompanionPane/src/main/res/layout/effect_control.xml:7]: this is a great refactor!

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks for the improvements Andrei!

**Pull request merged**